sortinghat/cli/client/client.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2014-2020 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     Santiago Dueñas <sduenas@bitergia.com>
+#
+
+import requests
+import uri
+
+from sgqlc.endpoint.http import HTTPEndpoint
+from sgqlc.operation import Operation
+
+from .schema import sh_schema
+
+
+class SortingHatClientError(Exception):
+    """SortingHat client error.
+
+    Generic exception raised by the client when an error is found
+    either with connection problems or with GraphQL queries.
+
+    :param msg: message error
+    :param errors: list of GraphQL errors; `None` when the error is
+        not related to GraphQL queries
+    """
+    def __init__(self, msg, errors=None):
+        self.msg = msg
+        self.errors = errors
+
+    def __str__(self):
+        return self.msg
+
+
+class SortingHatClient:
+    """SortingHat client.
+
+    This client allows to run operations in the SortingHat server
+    that listens in `host` and `port`.
+
+    After initializing an instance, call to `connect` to establish
```

Comment:
I think `call` should be replaced by `calls`.

Reply:
It's an imperative form in this case, so I think it's right. In any case, we can revisit the documentation after we make some progress.